includes/classes/traits/NotifierManager.php

```diff
@@ -94,6 +94,78 @@ function notify($eventID, $param1 = array(), &$param2 = null, &$param3 = null, &
         }
     }
 
+    /**
+     * Requires a define page, and calls regisistered Observers for content output.
+     *
+     * @param [type] $eventID
+     * @param array $param1
+     * @param [type] ...$params
+     * @return void
+     */
+    function insertContent($eventID, $param1 = array(), &...$params)
+    {
+        global $l;
```

Comment:
?

Reply:
Sorry, what's the question?  I experimented with using variable length argument lists instead of named $param1, $param2 etc.  Since you cannot strongly type the $paramx approach, varargs seems equally bad :)

Reply:
I didn't see where `$l` is used.

Reply:
Oh sorry, I use Monolog to expose a global `$l` logger so I can get sane logging out of ZenCart, this line slipped through the net.  It would be removed in a final version.